### PR TITLE
Fix #2266: [RTL] High-fi Admin and User PinVerification Screen

### LINKED ISSUE #2266: [RTL] High-fi AdminPinVerification [BLOCKED]
`AdminPinVerification` needs RTL support as shown in below screen.

PC AP (Empty) – RTL

Make sure that this works correctly in mobile (landscape + portrait) and tablet (landscape + portrait).

**Do not focus on keyboard, we are using native android keyboard.**

## References
Mobile Portrait: https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/grid/
Mobile Landscape: https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/grid/
Tablet: https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/grid/
Wiki: https://github.com/oppia/oppia-android/wiki/Working-on-UI

### PR DESCRIPTION
**Pull request by veena14cs**
## Explanation
This PR fixes #2266 . This PR implements new design for Admin and User Pin Verification
Older implementation of Pin Verification UI was using PinView library which was not supporting RTL. To fix this we removed this library and created new UI as per the new mocks.

## Old Screenshot LTR and RTL
Screenshot_1629793897........Screenshot_1629793938
Screenshot_1629793905........Screenshot_1629793945

## New Screenshot LTR and RTL
Screenshot_1633506133...Screenshot_1633506065

Screenshot_1633506118.....Screenshot_1633506083

Tablet

Screenshot_1633504178.....Screenshot_1633504220

Screenshot_1633504169.....Screenshot_1633504231

## Mock links
Mobile Portrait : https://xd.adobe.com/view/3dca36c2-5115-419c-b25e-0f10526b077c-6899/screen/4d274b3e-c27b-438a-8c74-c0ff16134080/
Mobile Landscape : https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/327723da-a1e8-40e4-aace-9c30d663b20b/
Tablet Portrait : https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/8de829a5-fc1c-4083-b510-44c1387ed9c2/
Tablet Landscape : https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/0648610b-7d70-403c-a42e-a206be75e85f/

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/build.gradle
Comment (rt4914) [inline app/build.gradle]: remove this from bazel files too.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@veena14cs Currently the mocks are used such that you have kept extreme right and left margin/padding as fixed as per mocks and the content inside is either `wrap` or `match`.

But instead the content should be fixed width and margin/padding should be variable.

For example if you see tablet mocks the content width is of `360dp`, so we should fix that and make margins variable. Similarly the `inputbox + show/hide` is `304dp` in mobile.

Also the background/toolbar is completely different as compared to mocks.

**Comment by veena14cs** [comment]
> @veena14cs Currently the mocks are used such that you have kept extreme right and left margin/padding as fixed as per mocks and the content inside is either `wrap` or `match`.
> 
> But instead the content should be fixed width and margin/padding should be variable.
> 
> For example if you see tablet mocks the content width is of `360dp`, so we should fix that and make margins variable. Similarly the `inputbox + show/hide` is `304dp` in mobile.

Shouldn't we keep variable width to support smaller devices?

**Comment by veena14cs** [comment]
> > @veena14cs Currently the mocks are used such that you have kept extreme right and left margin/padding as fixed as per mocks and the content inside is either `wrap` or `match`.
> > But instead the content should be fixed width and margin/padding should be variable.
> > For example if you see tablet mocks the content width is of `360dp`, so we should fix that and make margins variable. Similarly the `inputbox + show/hide` is `304dp` in mobile.
> 
> Shouldn't we keep variable width to support smaller devices?

I fixed the changes PTAL. Margins and width I have kept variable size to support all types of screen.

File: app/build.gradle
Comment (veena14cs) [reply app/build.gradle]: Removed library from bazel.

File: app/src/main/res/layout/pin_password_activity.xml
Comment (rt4914) [inline app/src/main/res/layout/pin_password_activity.xml]: Make this value `40` instead of `42dp`

File: app/src/main/res/layout/pin_password_activity.xml
Comment (rt4914) [inline app/src/main/res/layout/pin_password_activity.xml]: For different textviews you can use styles from `styles.xml` file.
For example for this textview you can change it too
```
<TextView
          style="@style/Heading1"
          android:id="@+id/hello_text_view"
          android:fontFamily="sans-serif-medium"
          android:text="@{String.format(@string/pin_password_hello, viewModel.profile.name)}"
          app:layout_constraintEnd_toEndOf="parent"
          app:layout_constraintHorizontal_bias="0.0"
          app:layout_constraintStart_toStartOf="parent"
          app:layout_constraintTop_toTopOf="parent" />
```

File: app/src/main/res/layout/pin_password_activity.xml
Comment (rt4914) [inline app/src/main/res/layout/pin_password_activity.xml]: Why does it need to be in `1` line only? I think it should be completely fine to expand it to mutliple lines as we do have space on this screen.

File: app/src/main/res/layout/pin_password_activity.xml
Comment (rt4914) [reply app/src/main/res/layout/pin_password_activity.xml]: Ditto, applies to other textviews in all files.

**Review by rt4914 — CHANGES_REQUESTED** [review]
For earlier [comment](https://github.com/oppia/oppia-android/pull/3684#pullrequestreview-732029890) the background/toolbar color changes have been missed. The entire screen is of yellow/cream color instead of blue color in mocks.

Also you can merge `pin_password_activity.xml` files by mentioning all values in dimens.xml file. 
For this follow these steps and it would become much easy:
1. On [text-compare](https://text-compare.com/) on left paste content from `mobile-portrait` file and on right paste from `mobile-landscape` and now for all difference create values in `dimens.xml` file and its other qualifier files.
2. Apply this same process for `mobile-portrait` vs `tablet-portrait` and `mobile-portrait` vs `tablet-portrait` too.

File: app/src/main/res/layout/pin_password_activity.xml
Comment (veena14cs) [reply app/src/main/res/layout/pin_password_activity.xml]: Done.

File: app/src/main/res/layout/pin_password_activity.xml
Comment (veena14cs) [reply app/src/main/res/layout/pin_password_activity.xml]: Done.

File: app/src/main/res/layout/pin_password_activity.xml
Comment (veena14cs) [reply app/src/main/res/layout/pin_password_activity.xml]: removed this line.

**Comment by veena14cs** [comment]
> For earlier [comment](https://github.com/oppia/oppia-android/pull/3684#pullrequestreview-732029890) the background/toolbar color changes have been missed. The entire screen is of yellow/cream color instead of blue color in mocks.
> 
> Also you can merge `pin_password_activity.xml` files by mentioning all values in dimens.xml file.
> For this follow these steps and it would become much easy:
> 
> 1. On [text-compare](https://text-compare.com/) on left paste content from `mobile-portrait` file and on right paste from `mobile-landscape` and now for all difference create values in `dimens.xml` file and its other qualifier files.
> 2. Apply this same process for `mobile-portrait` vs `tablet-portrait` and `mobile-portrait` vs `tablet-portrait` too.

I have fixed changes and updated screenshots in PR description.
 I think we cannot merge  `pin_password_activity.xml` files as mobile landscape screen is different. Rest all files are same.

**Comment by rt4914** [comment]
> > For earlier [comment](https://github.com/oppia/oppia-android/pull/3684#pullrequestreview-732029890) the background/toolbar color changes have been missed. The entire screen is of yellow/cream color instead of blue color in mocks.
> > Also you can merge `pin_password_activity.xml` files by mentioning all values in dimens.xml file.
> > For this follow these steps and it would become much easy:
> > 
> > 1. On [text-compare](https://text-compare.com/) on left paste content from `mobile-portrait` file and on right paste from `mobile-landscape` and now for all difference create values in `dimens.xml` file and its other qualifier files.
> > 2. Apply this same process for `mobile-portrait` vs `tablet-portrait` and `mobile-portrait` vs `tablet-portrait` too.
> 
> I have fixed changes and updated screenshots in PR description.
> I think we cannot merge `pin_password_activity.xml` files as mobile landscape screen is different. Rest all files are same.


@veena14cs Following changes are needed:
1. In `PinPasswordActivityPresenter` change the status bar color so that it can match mock. It might be doable by removing this line : `StatusBarColor.statusBarColorUpdate(R.color.pinInputStatusBar, activity, true)`
2. The shadow below toolbars is not visible. Check `add_profile_activity.xml` line `247` for reference.
3. There is one bug in this. Type 2 digits, click on show/hide and see how cursor shifts to start. This needs to be fixed.
4. For merging all xml files. Actually it is possible because the difference between mobile-portrait and mobile-landscape file is one line only which is `app:layout_constraintHorizontal_bias="0.0"` is part of `pin_password_input_pin` in portrait and it not available in landscape. This can be fixed by introducing `app:layout_constraintHorizontal_bias="0.5"` landscape also. Now we need to shift these values to `dimens` file which can be done with this code https://stackoverflow.com/a/8780360

**Comment by veena14cs** [comment]
> @veena14cs Following changes are needed:
> 
> 1. In `PinPasswordActivityPresenter` change the status bar color so that it can match mock. It might be doable by removing this line : `StatusBarColor.statusBarColorUpdate(R.color.pinInputStatusBar, activity, true)`
> 2. The shadow below toolbars is not visible. Check `add_profile_activity.xml` line `247` for reference.
> 3. There is one bug in this. Type 2 digits, click on show/hide and see how cursor shifts to start. This needs to be fixed.
> 4. For merging all xml files. Actually it is possible because the difference between mobile-portrait and mobile-landscape file is one line only which is `app:layout_constraintHorizontal_bias="0.0"` is part of `pin_password_input_pin` in portrait and it not available in landscape. This can be fixed by introducing `app:layout_constraintHorizontal_bias="0.5"` landscape also. Now we need to shift these values to `dimens` file which can be done with this code https://stackoverflow.com/a/8780360

@rt4914 thanks for the above reference it was really helpful. I have done changes to the list mentioned above.
1) Fixed statusbar.
2) Added shadow below toolbar.
3) Fixed bug of Edit text cursor on show/hide click event.
4) Merged `pin_password_activity.xml`

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

File: app/build.gradle
Comment (BenHenning) [reply app/build.gradle]: Please also remove from versions.bzl.

File: app/src/main/res/values/strings.xml
Comment (BenHenning) [inline app/src/main/res/values/strings.xml]: These need more context I think on where the text shows up. For example, the first text could be:

``input_pin_password_as_admin``

Please update all new string names accordingly to have more context.

**Review by BenHenning — CHANGES_REQUESTED** [review]
@veena14cs had a few comments. Also, could you please add screenshots for the 'before' screen to show more explicitly what changed?

Thanks.

File: app/src/main/res/values/strings.xml
Comment (veena14cs) [reply app/src/main/res/values/strings.xml]: Updated strings.

**Comment by veena14cs** [comment]
> @veena14cs had a few comments. Also, could you please add screenshots for the 'before' screen to show more explicitly what changed?
> 

Added before and after screenshots.

**Comment by veena14cs** [comment]
@prayutsu as we discussed in hangouts meeting and tried to solve maven update error. It wasn't working on Windows and as its on priority I would assign it to you to remove third party library and run this script using bazel command to update maven_install.json file.

**Comment by prayutsu** [comment]
> @prayutsu as we discussed in hangouts meeting and tried to solve maven update error. It wasn't working on Windows and as its on priority I would assign it to you to remove third party library and run this script using bazel command to update maven_install.json file.

@veena14cs Done.

File: .idea/misc.xml
Comment (BenHenning) [inline .idea/misc.xml]: Unrelated change--please revert.

**Review by BenHenning — CHANGES_REQUESTED** [review]
Thanks for adding the old screenshots @veena14cs.

Reviewed my codeowners files (except strings), but before I go further I have some fundamental concerns with this. @rt4914 we discussed that we should prefer keeping the existing user experience. Was a decision made somewhere to change the UX--did I possibly miss that? Also, was Chantel included in that decision & is the rationale documented? If so, could we please capture that in this PR. If not, we should make sure UX weighs in on this since it's a major user experience change.

**Comment by rt4914** [comment]
> Thanks for adding the old screenshots @veena14cs.
> 
> Reviewed my codeowners files (except strings), but before I go further I have some fundamental concerns with this. @rt4914 we discussed that we should prefer keeping the existing user experience. Was a decision made somewhere to change the UX--did I possibly miss that? Also, was Chantel included in that decision & is the rationale documented? If so, could we please capture that in this PR. If not, we should make sure UX weighs in on this since it's a major user experience change.

@BenHenning Yes we did finalise on changing the UX. @seanlip  and @mschanteltc are aware of this and the mocks were provided by Chantel. The main rationale was that actually there is no big preference from Chantel's side and she thinks either UI is fine and considering we were have a lot of technical difficulties fixing the previous UI we finally changed it. Also I have added you to that email thread today so you can have a look at the entire conversation.

File: app/build.gradle
Comment (veena14cs) [reply app/build.gradle]: Done.

File: .idea/misc.xml
Comment (veena14cs) [reply .idea/misc.xml]: Done.

**Comment by veena14cs** [comment]
> > Thanks for adding the old screenshots @veena14cs.
> > Reviewed my codeowners files (except strings), but before I go further I have some fundamental concerns with this. @rt4914 we discussed that we should prefer keeping the existing user experience. Was a decision made somewhere to change the UX--did I possibly miss that? Also, was Chantel included in that decision & is the rationale documented? If so, could we please capture that in this PR. If not, we should make sure UX weighs in on this since it's a major user experience change.
> 
> @BenHenning Yes we did finalise on changing the UX. @seanlip and @mschanteltc are aware of this and the mocks were provided by Chantel. The main rationale was that actually there is no big preference from Chantel's side and she thinks either UI is fine and considering we were have a lot of technical difficulties fixing the previous UI we finally changed it. Also I have added you to that email thread today so you can have a look at the entire conversation.


@BenHenning does this help in clarifying the changes? If yes can you please review this PR.

File: app/src/main/res/values/strings.xml
Comment (BenHenning) [inline app/src/main/res/values/strings.xml]: Should this be ``pin_password_toolbar_title``? It's a bit unclear on what the "what" is corresponding to this string.

**Review by BenHenning — COMMENTED** [review]
Thanks @rt4914 and @veena14cs. Yes, this helps clarify. Just had 1 comment, otherwise the PR LGTM. PTAL.

File: app/src/main/res/values/strings.xml
Comment (veena14cs) [reply app/src/main/res/values/strings.xml]: Changed it to `pin_password_toolbar_title`.

**Review by BenHenning — CHANGES_REQUESTED** [review]
Thanks @veena14cs. This LGTM from a code perspective (per my codeowner files), but it looks like CI is failing. In the very least, new dependencies need to be pinned for Maven. Can you please fix the CI failures & assign back?

**Comment by veena14cs** [comment]
> Thanks @veena14cs. This LGTM from a code perspective (per my codeowner files), but it looks like CI is failing. In the very least, new dependencies need to be pinned for Maven. Can you please fix the CI failures & assign back?

Thanks @BenHenning. To fix CI failures I need to run bazel in order to update Maven. @rt4914 can you please do it from your system and push the code as it is blocking me from submitting this PR.

**Comment by rt4914** [comment]
@BenHenning @veena14cs 
I am trying to follow these steps so as to fix the bazel issues.
1. Remove `pin-view` version from `versions.bzl` file.
2. Run `REPIN=1 bazel run @unpinned_maven//:pin` in terminal (It was successful)
3. Run `bazel run //scripts:generate_maven_dependencies_list -- $(pwd) third_party/maven_install.json scripts/assets/maven_dependencies.textproto scripts/assets/maven_dependencies.pb` in terminal.

I am getting following error:
Screenshot 2021-09-25 at 2 42 51 AM


After this I checked out to `develop` too and followed step 2 and 3 from above and I am still getting that same error.

@BenHenning Any idea what I am doing wrong?

File: app/src/main/java/org/oppia/android/app/profile/PinPasswordActivityPresenter.kt
Comment (BenHenning) [inline app/src/main/java/org/oppia/android/app/profile/PinPasswordActivityPresenter.kt:56]: Why is this needed now? It doesn't seem like we did it this way before.

**Review by BenHenning — COMMENTED** [review]
One comment @veena14cs, otherwise this is also a reminder to re-remove the pinview dependency from versions.bzl once we figure out what's going wrong with pinning.

> @BenHenning @veena14cs
> I am trying to follow these steps so as to fix the bazel issues.
> 
> 1. Remove `pin-view` version from `versions.bzl` file.
> 2. Run `REPIN=1 bazel run @unpinned_maven//:pin` in terminal (It was successful)
> 3. Run `bazel run //scripts:generate_maven_dependencies_list -- $(pwd) third_party/maven_install.json scripts/assets/maven_dependencies.textproto scripts/assets/maven_dependencies.pb` in terminal.
> 
> I am getting following error:
> Screenshot 2021-09-25 at 2 42 51 AM
> 
> After this I checked out to `develop` too and followed step 2 and 3 from above and I am still getting that same error.
> 
> @BenHenning Any idea what I am doing wrong?

@rt4914 are you able to build with Bazel locally? I.e. does ``bazel build //:oppia`` work? It looks like you have an Android SDK incompatibility with the Bazel environment, though I'm not clear yet on what the actual issue is.

**Comment by BenHenning** [comment]
Note that I'm building locally & will create a Gist with the file updates needed.

**Comment by BenHenning** [comment]
@veena14cs PTAL at https://gist.github.com/BenHenning/9e541aa731d5af2379d766a117ad3929/revisions & particularly the latest revision https://gist.github.com/BenHenning/9e541aa731d5af2379d766a117ad3929. That should contain the changes you need to remove the dependency & pass CI.

**Comment by veena14cs** [comment]
> @veena14cs PTAL at https://gist.github.com/BenHenning/9e541aa731d5af2379d766a117ad3929/revisions & particularly the latest revision https://gist.github.com/BenHenning/9e541aa731d5af2379d766a117ad3929. That should contain the changes you need to remove the dependency & pass CI.

@BenHenning I have updated files and also CI checks are passing now. PTAL

File: app/src/main/java/org/oppia/android/app/profile/PinPasswordActivityPresenter.kt
Comment (veena14cs) [reply app/src/main/java/org/oppia/android/app/profile/PinPasswordActivityPresenter.kt:56]: In older UI we were not having toolbar. In new version of Pin VIew UI toolbar is been introduced.

File: scripts/assets/maven_dependencies.textproto
Comment (BenHenning) [inline scripts/assets/maven_dependencies.textproto]: Please add EOF newline.

File: third_party/versions.bzl
Comment (BenHenning) [inline third_party/versions.bzl]: Why this change? It seems unrelated--I suggest reverting this part.

**Review by BenHenning — COMMENTED** [review]
Thanks @veena14cs. LGTM other than the two new comments.

File: scripts/assets/maven_dependencies.textproto
Comment (veena14cs) [reply scripts/assets/maven_dependencies.textproto]: Done.

File: third_party/versions.bzl
Comment (veena14cs) [reply third_party/versions.bzl]: Reverted.

**Comment by veena14cs** [comment]
@BenHenning all tests are failing in `PinPasswordActivityTest`

Screenshot (13)


I tested on develop too. It fails with the same error.

Screenshot (14)

**Comment by BenHenning** [comment]
@veena14cs did you try running that suite first on Robolectric? Per the CI logs, https://gist.github.com/BenHenning/fe5e145b8720e45f7f1afbf61e0af120 seem to be the actual blocking failures. I suggest trying to fix these first then retry the Espresso run.

That being said, the error you're seeing on Espresso shouldn't be happening per the initialization rule. Something definitely seems wrong there.

**Comment by veena14cs** [comment]
> @veena14cs did you try running that suite first on Robolectric? Per the CI logs, https://gist.github.com/BenHenning/fe5e145b8720e45f7f1afbf61e0af120 seem to be the actual blocking failures. I suggest trying to fix these first then retry the Espresso run.
> 
> That being said, the error you're seeing on Espresso shouldn't be happening per the initialization rule. Something definitely seems wrong there.

@BenHenning these tests were passing earlier and now I have added `land` qualifiers to pass this tests.

File: app/src/sharedTest/java/org/oppia/android/app/profile/PinPasswordActivityTest.kt
Comment (BenHenning) [inline app/src/sharedTest/java/org/oppia/android/app/profile/PinPasswordActivityTest.kt]: @rt4914 is landscape actually correct to use for these?

@veena14cs this might just be a case where the new UI isn't fitting within the screen on Robolectric & we now need more room in order for the test to pass. Does it pass with something like:

```kotlint
@Config(qualifiers = "w360dp-h760dp-port-xxxhdpi")
```

(I think that roughly corresponds to a Pixel XL)

**Review by BenHenning — COMMENTED** [review]
Thanks @veena14cs. The PR looks good, but I'm not sure about the test change. Changing to landscape seems like it might be a slight hack--maybe we can keep it in portrait but increase the size so that no scrolling is needed & all needed elements fit on the screen?

File: app/src/sharedTest/java/org/oppia/android/app/profile/PinPasswordActivityTest.kt
Comment (veena14cs) [reply app/src/sharedTest/java/org/oppia/android/app/profile/PinPasswordActivityTest.kt]: I did some changes in the UI and now tests passes without this.

**Comment by veena14cs** [comment]
> Thanks @veena14cs. The PR looks good, but I'm not sure about the test change. Changing to landscape seems like it might be a slight hack--maybe we can keep it in portrait but increase the size so that no scrolling is needed & all needed elements fit on the screen?

@BenHenning I have fixed the UI, may be somewhere it got changed while updating it with develop or while merging xml files into one and it went missing. Now, its fixed PTAL.

File: app/src/sharedTest/java/org/oppia/android/app/profile/PinPasswordActivityTest.kt
Comment (BenHenning) [reply app/src/sharedTest/java/org/oppia/android/app/profile/PinPasswordActivityTest.kt]: @rt4914 can you re-review the layout changes & approve if they look good to you? Test changes seem good to me.

File: app/src/main/res/layout/pin_password_activity.xml
Comment (BenHenning) [inline app/src/main/res/layout/pin_password_activity.xml]: Please add EOF newline.

**Review by BenHenning — COMMENTED** [review]
Thanks @veena14cs. PR generally LGTM, but I think Rajat should take one more pass since there seemed to be a lot of layout changes in the last commit.

File: app/src/main/res/layout/pin_password_activity.xml
Comment (veena14cs) [reply app/src/main/res/layout/pin_password_activity.xml]: Done.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@veena14cs The UI does not look correct.
1. The label `Administrator's 5 Digit PIN` UI is different from that of `admin_auth_activity`.
2. In landscape mode the UI does not look in centre.
3. Toolbar shadow is not correct.
Screenshot_1633438399


Also, suggest updating the screenshots in PR Description

**Comment by veena14cs** [comment]
> @veena14cs The UI does not look correct.
> 
> 1. The label `Administrator's 5 Digit PIN` UI is different from that of `admin_auth_activity`.
> 2. In landscape mode the UI does not look in centre.
> 3. Toolbar shadow is not correct.
>    Screenshot_1633438399
> 
> Also, suggest updating the screenshots in PR Description

@rt4914 I have addressed and updated the changes
1. Changed the UI keeping it consistent with `admin_auth_activity`.
2. Centered UI at the center in landscape mode.
3. Corrected Toolbar shadow in Tablet mode.

Updated screenshots in PR description. PTAL Thanks.

File: app/src/sharedTest/java/org/oppia/android/app/profile/PinPasswordActivityTest.kt
Comment (BenHenning) [reply app/src/sharedTest/java/org/oppia/android/app/profile/PinPasswordActivityTest.kt]: Marking as resolved since Rajat is taking a look.

**Review by BenHenning — APPROVED** [review]
Thanks @veena14cs. This LGTM; deferring to Rajat for final approval.

@rt4914 please re-add me if anything changes from the build or testing side.

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**
